packages/block-library/src/gallery/deprecated.js

```diff
@@ -19,6 +20,183 @@ import {
 } from './constants';
 
 const deprecated = [
+	{
+		attributes: {
+			images: {
+				type: 'array',
+				default: [],
+				source: 'query',
+				selector: '.blocks-gallery-item',
+				query: {
+					url: {
+						type: 'string',
+						source: 'attribute',
+						selector: 'img',
+						attribute: 'src',
+					},
+					fullUrl: {
+						type: 'string',
+						source: 'attribute',
+						selector: 'img',
+						attribute: 'data-full-url',
+					},
+					link: {
+						type: 'string',
+						source: 'attribute',
+						selector: 'img',
+						attribute: 'data-link',
+					},
+					alt: {
+						type: 'string',
+						source: 'attribute',
+						selector: 'img',
+						attribute: 'alt',
+						default: '',
+					},
+					id: {
+						type: 'string',
+						source: 'attribute',
+						selector: 'img',
+						attribute: 'data-id',
+					},
+					caption: {
+						type: 'string',
+						source: 'html',
+						selector: '.blocks-gallery-item__caption',
+					},
+				},
+			},
+			ids: {
+				type: 'array',
+				items: {
+					type: 'number',
+				},
+				default: [],
+			},
+			columns: {
+				type: 'number',
+				minimum: 1,
+				maximum: 8,
+			},
+			caption: {
+				type: 'string',
+				source: 'html',
+				selector: '.blocks-gallery-caption',
+			},
+			imageCrop: {
+				type: 'boolean',
+				default: true,
+			},
+			linkTo: {
+				type: 'string',
+			},
+			sizeSlug: {
+				type: 'string',
+				default: 'large',
+			},
+		},
+		supports: {
+			anchor: true,
+			align: true,
+		},
+		isEligible( { images } ) {
+			return (
+				images &&
+				images.some(
+					( image ) => image.alt !== __unstableStripHTML( image.alt )
+				)
+			);
+		},
+		migrate( attributes ) {
+			return {
+				...attributes,
+				images: attributes.images.map( ( image ) => {
+					if ( image.alt !== __unstableStripHTML( image.alt ) ) {
+						return __unstableStripHTML( image.alt );
+					}
+					return image.alt;
```

Comment:
We need to return the entire image object (with the modified/unmodified `alt`) here, no? :smile:

Reply:
We absolutely, 100% do! 😅

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/26574/commits/82c2a6f0aecf54d9ebd75343d0c25cab555b102b